automated-comments/go/hamming/use_of_naked_returns.md

```diff
@@ -0,0 +1,4 @@
+`Naked returns` (or `bare returns`) are considered evil by most of the Go community. Simply because they make code harder to read.
+The official [Go code review guide](https://github.com/golang/go/wiki/CodeReviewComments#naked-returns) suggests to avoid them if the function has more than a few lines of code.
+There is also a [proposal](https://github.com/golang/go/issues/21291) to remove bare returns altogether in Go 2.
+My suggestin would be to avoid them.
```

Comment:
> are considered evil by most of the Go community

That's a very strong statement. How about:

> The Go community recommends avoiding _naked returns_ (or _bare returns_) under normal circumstances, as they typically make code harder to read. See the [official Go code review guide](https://github.com/golang/go/wiki/CodeReviewComments#naked-returns) for details.
> There is a [proposal](https://github.com/golang/go/issues/21291) to remove bare returns altogether in Go 2.

Reply:
Was it intentional that you included the `official` in the link?